Sources/Apodini/Components/Codable.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Codable.swift
+//  
+//
+//  Created by Tim Gymnich on 22.12.20.
+//
+
+import Foundation
+import NIO
+
+
+extension EventLoopFuture: Encodable where Value: Encodable {
+    public func encode(to encoder: Encoder) throws {
+        let encodable = try self.wait()
```

Comment:
This would still block the thread right? 

I sketched my idea as described above: https://github.com/Apodini/Apodini/compare/async-encodable-eventloopfuture

I isn't fully cleaned up, and it still lacks implementation using that part in the RequestHandler. (ResponseTransformer where never made for this I think, at least they would have been passed a EventLoopFuture and I would guess this wasn't intended?). 

ut something like this should work to keep it non blocking. And a similar approach is currently used for the implementation of `Action<...>`, meaning a clean implementation is possible for sure.